source/src_io/dos.cpp

```diff
@@ -157,10 +158,15 @@ bool Dos::calculate_dos
 {
 	ModuleBase::TITLE("Dos","calculae_dos");
 	std::ofstream ofs;
+	std::ofstream ofs1;
 	if(GlobalV::MY_RANK==0)
 	{
 		ofs.open(fa.c_str());//make the file clear!!
+		ofs1.open(fa1.c_str());//make the file clear!!
 	}
+	std::vector<double> dos;
+	std::vector<double> ene;
+	std::vector<double> dos2; //dos_smearing
```

Comment:
You are using both '1' and '2' in variable names for smearing file output. Using "dos_smering" or something like that instead of numbers will be better.

Reply:
sounds good